src/www/ui/admin-dashboard.php

```diff
@@ -159,7 +196,6 @@ function DatabaseMetrics()
     $connection_count = $row['val'];
 
     /**** Active connection count ****/
-    $current_query = (strcmp($this->pgVersion['server'], "9.2") >= 0) ? "state" : "current_query";
```

Comment:
This functionality is already implemented in line 222 (Didn't see that either)
![image](https://user-images.githubusercontent.com/6639323/27331963-2e426a26-55c0-11e7-831c-f661e2a9398c.png)